### PR TITLE
Origin/input

### PR DESCRIPTION
**Pull request by DenyGurzhiy**
Hello, According to last online meeting on 07/11, I've made some major correction to Viktoria's work (We agreed about this).
As Tamara suggested, I've made this pull request against Victoria's one for future merging.
Also:
1. I haven't deleted part of HTML and CSS code for search input, but I've done unavailable it (Probable it could be usable for other team who provide creating of that kind of input).
2. Links to SVG was changed on files that I've put to separate folder: components/icons

### CODE REVIEW

**Review by TatianaBortnik — APPROVED** [review]
Looks good

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
...
+```
+npm start
+```
+
+### How to stop server
+Key combination: `Ctrl + C`
+
+### Links
+Home page of main application: `http://localhost:3000`
+Help page to see list of components: `http://localhost:3000/components`
```
Comment (tbazko) [inline README.md:35]: @DenyGurzhiy why is this in your MR?

File: client/components/input/input.css [hunk 2]
```diff
@@ -1,117 +1,98 @@
...
+/* .search:has(input:enabled:valid),
+.search:has(input:enabled:placeholder-shown:valid),
+.search:has(input:disabled),
+.search:disabled {
+  max-width: 430px;
+  background: var(--primary-white);
+  background-image: url("data:image/svg+xml;charset=UTF-8,%3csvg width='26' height='26' viewBox='0 0 26 26' fill='none' xmlns='http://www.w3.org/2000/svg'%3e%3cpath fill-rule='evenodd' clip-rule='evenodd' d='M0 10.8181C0 4.84344 4.84344 2.38419e-07 10.8181 0H10.8183C16.7926 0.000757515 21.6362 4.84347 21.6362 10.8181C21.6362 13.4454 20.6997 15.8539 19.1423 17.728L25.7071 24.2929C26.0977 24.6834 26.0977 25.3166 25.7071 25.7071C25.3166 26.0976 24.6834 26.0976 24.2929 25.7071L17.7281 19.1422C15.8539 20.6997 13.4454 21.6362 10.8181 21.6362C4.84344 21.6362 2.38419e-07 16.7928 0 10.8181ZM10.818 2C5.94794 2.00007 2 5.94805 2 10.8181C2 15.6882 5.94801 19.6362 10.8181 19.6362C15.6882 19.6362 19.6362 15.6882 19.6362 10.8181C19.6362 5.94831 15.6883 2.00069 10.8181 2' fill='%230061FF'/%3e%3c/svg%3e");
+  background-repeat: no-repeat;
+  background-position: 95% 50%;
+} */
```
Comment (tbazko) [inline client/components/input/input.css:25]: Don't leave this around. Nobody will ever find it here even if they need smth like this. This is essentially dead code.

**Review by tbazko — APPROVED** [review]
@DenyGurzhiy You can merge this yourself after you remove commented code.

**Comment by DenyGurzhiy** [comment]
I've deleted all commented code.

**Pull request merged**